docs/testing/continuous-integration.md

```diff
@@ -22,63 +22,42 @@ in development. Except when working on the CI configuration itself, a
 developer should never have to repeatedly wait 10 minutes for a full CI
 run to iteratively debug something.
 
-## CircleCI
+## GitHub Actions
 
 ### Useful debugging tips and tools
 
 * Zulip uses the `ts` tool to log the current time on every line of
-the output in our CircleCI scripts.  You can use this output to
+the output in our GitHub Action scripts.  You can use this output to
 determine which steps are actually consuming a lot of time.
 
-* You can [sign up your personal repo for CircleCI][circleci-setup] so
-that every remote branch you push will be tested, which can be helpful
-when debugging something complicated.
+* GitHub Actions runs on every branch you push on your Zulip fork.
+This is helpful when debugging something complicated.
 
-* With your personal repo signed up, CircleCI
-[allows you to SSH][circleci-ssh] into the job container if a job
-fails. SSHing into the containers can be helpful, especially in rare
+* You can also ssh into a container to debug failures.
+SSHing into the containers can be helpful, especially in rare
 cases where the tests are passing in your computer but failing in the
-CI. Make sure that you have uploaded your SSH keys to GitHub: CircleCI
-uses those SSH keys for authentication.
-
-[docker-hub]: https://hub.docker.com/
-[circleci-setup]: ../git/cloning.html#step-3-configure-continuous-integration-for-your-fork
-[circleci-ssh]: https://circleci.com/docs/2.0/ssh-access-jobs/
+CI. There are various [Actions](https://github.com/marketplace?type=actions&query=debug+ssh)
+available on GitHub Marketplace to help you SSH into a container. They
+use different methods to SSH into the container. Use the one you find
+easy to setup.
 
 ### Suites
 
-The main CircleCI configuration file defining how the tests are run is
-[./circleci/config.yml][circleci-config].  Our code for running the
-tests in CI lives under `tools/ci`; but they are mostly thin wrappers
-around [Zulip's test suites](../testing/testing.md) or production
-installer tooling.
-
-[circleci-config]: https://github.com/zulip/zulip/blob/master/.circleci/config.yml
-
-We run multiple jobs during a CircleCI build to run Zulip's test
-suites on our supported production platforms. They are currently:
-
-* bionic-backend-frontend
-* focal-backend
-
-Each runs the Zulip backend test suites, using the indicated
-platform/OS.  As suggested by the names, only one
-suite runs the frontend test suites, since those are not
-platform-dependent.
-
-Additionally, there a couple jobs designed to do an end-to-end test on
-Zulip's production installer:
+We run multiple jobs during a GitHub Actions build to run Zulip's test
+suites on our supported production platforms. See [Actions tabs](https://github.com/zulip/zulip/actions)
+for full list of Actions that we run.
 
-* bionic-production-build
-* bionic-production-install
-* xenial-legacy
```

Comment:
I think this deletes maybe more than we ideally want to about what we test, mainly because some "actions" have multiple "jobs" as well.

Reply:
Added new job details.